### ISSUE TITLE
binary/binary - metrics from confusion matrix

### ISSUE THREAD

**Issue by aaweaver-actuary** [aaweaver-actuary/sb-eda#4]
1. true positive (recall)
2. false positive
3. precision
4. f1 score
5. accuracy
6. specificity
7. AUC
8. ROC curve

```python
from sklearn.metrics import confusion_matrix, precision_score, recall_score, f1_score, accuracy_score, roc_auc_score, roc_curve

# assume already trained `xgb` model and have `X_test` and `y_test`
preds = xgb.predict(X_test)
cm = confusion_matrix(y_test, preds)

true_positive = cm[1][1]
false_positive = cm[0][1]
precision = precision_score(y_test, preds)
recall = recall_score(y_test, preds)
f1 = f1_score(y_test, preds)
accuracy = accuracy_score(y_test, preds)
specificity = true_negative / (true_negative + false_positive)
auc = roc_auc_score(y_test, preds)
fpr, tpr, thresholds = roc_curve(y_test, preds)
```
